internal/client/gitea.go

```diff
@@ -51,6 +51,36 @@ func NewGitea(ctx *context.Context, token string) (Client, error) {
 	return &giteaClient{client: client}, nil
 }
 
+// CloseMilestone closes a given milestone.
+func (c *giteaClient) CloseMilestone(ctx *context.Context, repo Repo, title string) error {
+	milestone, err := c.getMilestoneByTitle(repo, title)
+
+	if err != nil {
+		return err
+	}
+
+	if milestone == nil {
+		return fmt.Errorf("no open milestone matching: %s", title)
```

Comment:
maybe create an error type `ErrNoMilestoneFound` or something like that? 🤔

Reply:
Updated in https://github.com/goreleaser/goreleaser/pull/1657/commits/5aea6dd9fed3c81f9878bae771456b73722ba4a7 👍